src/JBrowse/Store/SeqFeature/BigBed.js

```diff
@@ -0,0 +1,87 @@
+define( [
+            'dojo/_base/declare',
+            'dojo/_base/lang',
+            'dojo/_base/array',
+            './BigWig',
+            './BigWig/Window',
+            'JBrowse/Model/SimpleFeature'
+        ],
+        function(
+            declare,
+            lang,
+            array,
+            BigWig,
+            Window,
+            SimpleFeature
+        ) {
+return declare(BigWig,
+
+ /**
+  * @lends JBrowse.Store.SeqFeature.BigBed
+  */
+{
+    constructor: function(args) {
+        if(args) {
+            this.groupFeatures = args.groupFeatures;
+        }
+    },
+
+    _getFeatures: function( query, featureCallback, endCallback, errorCallback ) {
+
+        var chrName = this.browser.regularizeReferenceName( query.ref );
+        var min = query.start;
+        var max = query.end;
+
+        var v = this.getView();
+
+        if( !v ) {
+            endCallback();
+            return;
+        }
+
+        v.readWigData( chrName, min, max, (features) => {
+            if(this.groupFeatures) {
+                var genes = {};
+                array.forEach( features || [], function(feature) {
+                    var id = feature.get('geneId');
+                    if(!genes[id]) {
+                        genes[id] = new SimpleFeature({
+                            id: feature.get('geneId'),
+                            data: {
+                                start: feature.get('start'),
+                                end: feature.get('end'),
+                                type: 'gene',
+                                name: feature.get('geneName'),
+                                id: feature.get('id'),
+                                subfeatures: []
+                            }
+                        });
+                    }
+                    genes[id].data.subfeatures.push(feature);
+                });
+                Object.keys(genes).forEach(function(name) {
+                    featureCallback(genes[name]);
+                });
+            }
```

Comment:
`groupFeatures` is never going to be robust, because unlike the gff3tabix, there is no parent feature that gives you what span you will need to fetch to get all the subfeatures.  I think we should probably remove `groupFeatures`.

Reply:
I guess that's true. Unless longest transcript over a region could be used as a proxy for that. I guess remove for now is good though